CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Relaxed ``cartridge pack`` `--version` and `--suffix` verification,
```

Comment:
To be honest, I'm not sure "relax" is the right word. "Untighten" or "loose" sounds more suitable.

Reply:
Changed